ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ReportJobs.tsx

```diff
@@ -58,22 +60,32 @@ type ReportJobsProps = {
 
 function ReportJobs({ scanConfig }: ReportJobsProps) {
     const { page, perPage, setPage, setPerPage } = useURLPagination(10);
-    const [filteredStatuses, setFilteredStatuses] = useState<RunState[]>([]);
-    const [showOnlyMyJobs, setShowOnlyMyJobs] = React.useState<boolean>(false);
+    const { searchFilter, setSearchFilter } = useURLSearch();
 
-    const onReportStatusFilterChange = (_checked: boolean, selectedStatus: RunState) => {
-        setFilteredStatuses((prevFilteredStatuses) => {
-            const isStatusIncluded = prevFilteredStatuses.includes(selectedStatus);
-            if (isStatusIncluded) {
-                return prevFilteredStatuses.filter((status) => status !== selectedStatus);
-            }
-            return [...prevFilteredStatuses, selectedStatus];
-        });
+    const filteredReportStates = ensureStringArray(searchFilter['Report state']);
+    const viewOnlyMyJobs = ensureBoolean(searchFilter['View only my jobs']);
```

Comment:
Remind me what proportion of the time to read `searchFilter` key like `'View only my jobs'` as search key in API payload versus for conditional rendering of page?

The reason I ask is because Title Case seems like the convention for search key in API payload:
https://github.com/stackrox/stackrox/blob/master/pkg/search/options.go

Versus camelCase like `'scanConfigTab'` for conditional rendering.

Reply:
I see what you mean. By naming it something like `?viewOnlyMyJobs=[true/false]`, it separates this from the searchFilter parameters, which are typically used for querying data in the backend. Since this is more of a UI-specific control and not a search filter criteria, it makes sense to handle it separately in the URL. This would keep things clean and avoid potential conflicts with API query params, while still allowing us to control it through the URL.